wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageAction.swift

```diff
@@ -21,7 +21,8 @@ import WireDataModel
 import UIKit
 
 enum MessageAction: CaseIterable, Equatable {
-    static var allCases: [MessageAction] = [.digitallySign,
+    static var allCases: [MessageAction] = [.visitLink("https://google.com"),
```

Comment:
why google?

Reply:
It doesn't matter. Any valid url would do.

Reply:
Let me make that wire.com :D